tests/test_core/test_alignment.py

```diff
@@ -2571,6 +2571,31 @@ def test_add_from_ref_aln(self):
             ValueError, aln1.add_from_ref_aln, aln2_wrong_refseq
         )  # test wrong_refseq
 
+    def test_annotate_matches_to(self):
+        """annotate_matches_to method should attach
+         annotations correctly to a Sequence object."""
```

Comment:
change this docstring to """Aligned. annotate_matches_to correctly delegates to sequence"""